sound/soc/sof/ipc4-alh.h

```diff
@@ -0,0 +1,50 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause) */
+/*
+ * This file is provided under a dual BSD/GPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ *
+ * Copyright(c) 2021 Intel Corporation. All rights reserved.
+ *
+ * Author: Bard Liao <yung-chuan.liao@linux.intel.com>
+ */
+
+#ifndef __SOF_INTEL_IPC4_ALH_H
+#define __SOF_INTEL_IPC4_ALH_H
+
+#define ALH_MAX_NUMBER_OF_GTW   1
+
+struct sof_gtw_attributes {
+	// Gateway data requested in low power memory.
+	uint32_t lp_buffer_alloc : 1;
+
+	//Gateway data requested in register file memory.
+	uint32_t alloc_from_reg_file : 1;
+
+	// Reserved field
+	uint32_t _rsvd : 30;
+} __packed;
```

Comment:
this is a common definition if I am not mistaken, it shouldn't be part of ALH support only.